test.html

```diff
@@ -1,13 +1,15 @@
-<!doctype html>
+<!DOCTYPE html>
```

Comment:
Yeah this is a common misconception. Under XHTML parsing rules the "DOCTYPE" keyword *must* be uppercase, but under HTML (and modern HTML5) parsing rules, it's case-insensitive (see https://dev.w3.org/html5/html-author/#html-and-xhtml-comparison)